src/ast.js

```diff
@@ -861,6 +861,20 @@ class AstSerializer {
 		if(htmlAttribute) {
 			let fn = ModuleScript.evaluateAttribute(htmlAttribute, this.filePath);
 			let context = Object.assign({}, this.helpers, options.componentProps, this.globalData);
+
+			// Check the string we're evaulating for any references to context.
+			const propertyNames = Array.from(htmlAttribute.matchAll(/this\.([a-zA-Z_$][0-9a-zA-Z_$]+)/gmi), (m) => m[1]);
```

Comment:
This regex doesnt cover other unicode characters that are technically allowed in property names but are unlikely to be used. Ideally you'd want to avoid regex and parse this into an AST of itself...